packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -875,6 +892,70 @@ Accounts.prototype.signTransactionWithSignature = function signTransactionWithSi
     });
 };
 
+/**
+ * combineSignatures combines RLP encoded raw transaction strings.
+ * combineSignatures compares transaction before combining, and if values in field are not same(except default value), this throws error.
```

Comment:
Don't we need the description about the fee payer?
It seems that it is okay even if the fee payer is empty.

Reply:
I added descriptions for fee payer above of that logic. 
But i think, feePayer can't be empty when fee delegated transaction. 
If someone try to encode themselves to skip feePayer in fee delegated transaction, then decode result will be different.

But i can add description for feePayer key field about default value in here again.

Reply:
I added descriptions about default value of fee payer. :)